src/questions/derivatives.yaml

```diff
@@ -0,0 +1,948 @@
+  hint: NoHint
+  explanation: Low liqiuidity in forward markets refers to difficulty while transaction are taking place.
+  answerKeys:
+    - C
+  subTopics:
+    - "types-of-derivatives"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Investors face losses
+      key: A
+    - content: Conversion to cash is difficult 
+      key: B
+    - content: Transactions are not carried out easily
+      key: C
+    - content: None of the above
+      key: D
+- uuid: 82d71f82-8e88-43ff-9d13-1e3cb371d286
+  type: SingleChoice
+  content: Which of the following are the disadvantages of forward contract?
+  hint: NoHint
+  explanation: All the reasons mentioned constitute to the disadvantges of a forward contract.
+  answerKeys:
+    - D
+  subTopics:
+    - "types-of-derivatives"
+  difficultyLevel: "Low"
+  choices:
+    - content: Finding a counterparty is difficult
+      key: A
+    - content: Low liquidity
+      key: B
+    - content: Default risk
+      key: C
+    - content: All of the above
+      key: D
+- uuid: ad133f59-6685-42fe-b47f-d2d575140e1c
+  type: SingleChoice
+  content: How are forward contracts better over futures?
+  hint: Benefits of Forward and Futures contract
+  explanation: Forward and Futures contract are identical in temrns of standardisation, default risk and they are equally liquid as well.
+  answerKeys:
+    - D
+  subTopics:
+    - "types-of-derivatives"
+  difficultyLevel: "Medium"
+  choices:
+    - content: They are standardised
+      key: A
+    - content: They have lower default risk
+      key: B
+    - content: They are more liquid
+      key: C
+    - content: None of the above
+      key: D
+- uuid: f4abb691-9853-425c-b241-f72cfebfa277
+  type: SingleChoice
+  content: Futures are regularly traded on
+  hint: NoHint
+  explanation: Chicago Board of Trade generally trades in futures contract.
+  answerKeys:
+    - A
+  subTopics:
+    - "types-of-derivatives"
+  difficultyLevel: "Low"
+  choices:
+    - content: Chicago Board of Trade
+      key: A
+    - content: New York Stock Exchange
+      key: B
+    - content: American Stock Exchange
+      key: C
+    - content: Chicago Board of Options Exchang
+      key: D
+- uuid: d21489fc-e65d-4bca-981b-d305bf208a22
+  type: MultipleChoice
+  content: What are the effects to hedge on futures market?
+  hint: Hedging reduces the risk
+  explanation: Hedging in futures market reduces the chances of gain and losses.
+  answerKeys:
+    - A
+    - B
+  subTopics:
+    - "types-of-derivatives"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Reduces the gain chnaces
+      key: A
+    - content: Reduce the loss chances
+      key: B
+    - content: High portfolio earning
+      key: C
+    - content: All of the above
+      key: D
+- uuid: 051125c0-74d8-463e-a23c-79d251a58c5a
+  type: SingleChoice
+  content: What happens to a bank that has perfectly hedged its portfolio during the time when interest rates fall in the futures market?
```

Comment:
We need to update the question as `interest rate` is not a very contextual term in Futures

Reply:
Done